requirements.txt

```diff
@@ -10,4 +8,3 @@ prometheus_client
 python-json-logger
 jupyterhub
 jsonschema
-pycurl
```

Comment:
I was expecting to see this line be replaced with an include/reference to another requirements file to make sure pycurl continues to be installed. What did I miss?

Reply:
right, I would expect that too and now I am confused.

`pycurl` is installed in `helm-chart/images/binderhub/requirements.txt`
https://github.com/jupyterhub/binderhub/blob/0b5080ade21dbda5ad159e675bacf25c61d8e422/helm-chart/images/binderhub/requirements.txt#L1
And when I go though the docs, ["User interface changes"](https://github.com/jupyterhub/binderhub/blob/master/CONTRIBUTING.md#user-interface-changes) is the only place that we only install `requirements.txt` and there we don't need to have pycurl. For example, when we run tests locally, we have to install `dev-requirements.txt` which installs also `helm-chart/images/binderhub/requirements.txt`.